backend/app/services/subscription/market_service.py

```
@@ -0,0 +1,651 @@
+            sort_by: Sort by 'rental_count' or 'recent'
+            search: Optional search query for name/description
+            skip: Pagination offset
+            limit: Pagination limit
+
+        Returns:
+            Tuple of (list of MarketSubscriptionDetail, total count)
+        """
+        # Query all active subscriptions
+        query = db.query(Kind).filter(
+            Kind.kind == "Subscription",
+            Kind.is_active == True,
```

Ruff (0.14.13)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)